screwdriver.yaml

```diff
@@ -0,0 +1,105 @@
+shared:
+  image: node:12
+  environment:
+    SKOPEO_PACKAGE: skopeo
+    ZSTD_PACKAGE: zstd
+
+jobs:
+  skopeo-build:
+    requires: [~pr, ~commit]
+    image: centos:centos7
+    steps:
+      - make: |
+          yum install -y epel-release
+          yum install -y wget git make gcc gcc-c++ jq
+          wget -q -O go1.15.6.tar.gz https://golang.org/dl/go1.15.6.linux-amd64.tar.gz
```

Comment:
`1.15.6` is hard-coded here.

Reply:
yeah, that's the version we use in launcher